### PR TITLE
Update readme with a quick start

### PR DESCRIPTION
**Pull request by wayne-li2**
Optional merge -- as someone who's a sqlite beginner, this would have been helpful.

I know it doesn't cover everything (prepared statements, for example) and only covers pure Dart for now.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,4 +7,75 @@ The main package in this repository is [`sqlite3`](sqlite3), which contains all
...
+  );
+''';
+db.execute(createTableStatement);
+
+// Insert a row.
+String insertRowStatement = '''
+  INSERT INTO dog (name, breed)
+  VALUES ("Doggo", "Golden Retriever");
+''';
+db.execute(insertRowStatement);
```
Comment (simolus3) [inline README.md:32]: Maybe we can use

```suggestion
// Use a prepared statement to insert a row with values
PreparedStatement insertRowStatement = db.prepare('''
  INSERT INTO dog (name, breed)
  VALUES (?, ?);
''');
insertRowStatement.execute(["Doggo", "Golden Retriever"]);
```

Then we have an example for prepared statements too :)

File: README.md [hunk 2]
```diff
@@ -7,4 +7,75 @@ The main package in this repository is [`sqlite3`](sqlite3), which contains all
 In can be used both in Flutter apps or in standalone Dart applications.
 
 The `sqlite3_flutter_libs` and `sqlcipher_flutter_libs` packages contain no Dart code at all. Flutter users can depend
-on one of them to include native libraries in their apps.
\ No newline at end of file
+on one of them to include native libraries in their apps.
+
+## Quick Start (pure Dart)
+
+```
```
Comment (simolus3) [inline README.md:14]: For better syntax highlighting, let's make this
```suggestion
```dart
```

File: README.md [hunk 3]
```diff
@@ -7,4 +7,75 @@ The main package in this repository is [`sqlite3`](sqlite3), which contains all
...
+
+// Insert multiple rows.
+String insertRowsStatement = '''
+  INSERT INTO dog (name, breed)
+  VALUES
+    ("Doggo", "Golden Retriever"),
+    ("Doggie", "Chihuahua"),
+    ("Doggy", "Dalmatian");
+''';
+db.execute(insertRowsStatement);
```
Comment (simolus3) [inline README.md:67]: If we replace the first insert with a `PreparedStatement`, maybe we can use it here too:

```suggestion
insertRowStatement
  ..execute(["Doggo", "Golden Retriever"])
  ..execute(["Doggie", "Chihuahua"])
  ..execute(["Doggy", "Dalmatian"]);

insertRowStatement.dispose();
```

File: README.md [hunk 4]
```diff
@@ -7,4 +7,75 @@ The main package in this repository is [`sqlite3`](sqlite3), which contains all
...
+
+resultSet = db.select("SELECT * FROM dog;");
+print(resultSet);
+
+// Use the result set and iterate through the result rows.
+Iterator<Row> resultSetIterator = resultSet.iterator;
+while (resultSetIterator.moveNext()) {
+  Row row = resultSetIterator.current;
+  print('Dog Object: [id: ${row['dog_id']}, name: ${row['name']}, breed: ${row['breed']}]');
+}
```
Comment (simolus3) [inline README.md:77]: Typically we use a `for` loop to implicitly create an iterator:
```suggestion
for (Row row in resultSet) {
  print('Dog Object: [id: ${row['dog_id']}, name: ${row['name']}, breed: ${row['breed']}]');
}
```

**Comment by wayne-li2** [comment]
Wow, no I didn't see the example! I'm coming from another language and totally new to Dart, so navigating around pub is still kind of new to me.

Usually, I go straight for the README, so I didn't see the example file. Let me add a link to the example file, which I think covers much of this already. :) 

Thank you for taking the time to help me!

**Comment by wayne-li2** [comment]
All I did now was add a result set example to main.dart. Otherwise I added a relative link from the README to the example file.

Let me know if there's anything else you want me to tackle.

File: sqlite3/example/main.dart [hunk 5]
```diff
@@ -29,7 +29,17 @@ void main() {
 
   // You can run select statements with PreparedStatement.select, or directly
   // on the database:
-  print(db.select('SELECT * FROM artists WHERE name LIKE ?', ['The %']));
+  final ResultSet resultSet =
+      db.select('SELECT * FROM artists WHERE name LIKE ?', ['The %']);
+
+  // You can iterate on the result set multiple ways to retrieve Row objects
```
Comment (simolus3) [inline sqlite3/example/main.dart:35]:
```suggestion
  // You can iterate on the result set in multiple ways to retrieve Row objects
```

Comment (wayne-li2) [reply sqlite3/example/main.dart]: Done.

**Comment by simolus3** [comment]
Thanks!

**Pull request merged**